src/antares/model/study.py

```diff
@@ -256,9 +256,9 @@ def create_area(
         self._areas[area.id] = area
         return area
 
-    def delete_area(self, area: Area) -> None:
-        self._area_service.delete_area(area)
-        self._areas.pop(area.id)
+    def delete_area(self, area_id: str) -> None:
```

Comment:
Here I think we should still have an Area as an argument and then pass the id. It would be clearer for the user I believe